hosts/linux/unsigned-int64/networking/wireguard/wireguard0.nix

```diff
@@ -1,78 +1,60 @@
-{ config, pkgs, lib, ... }:
-
+{ config, pkgs, lib, agenix, ... }:
+let
+  ip = "${pkgs.iptables}/bin/iptables";
+  ip6 = "${pkgs.iptables}/bin/ip6tables";
+in
 {
   networking = {
     wg-quick.interfaces = {
       wireguard0 = {
-        address = [ "192.168.10.1/24" "dced:2718:5f06:718a::1/64" "10.64.10.1/24" "fd02:f8eb:7ca4:5f4c::1/64"];
+        address = [ "192.168.10.1/24" "dced:2718:5f06:718a::1/64" "10.64.10.1/24" "fd02:f8eb:7ca4:5f4c::1/64" ];
         listenPort = 51820;
-        privateKeyFile = "";
-
+        privateKeyFile = config.age.secrets.wireguard-server.path
         postUp = ''
-          ${pkgs.iptables}/bin/iptables -A FORWARD -i wireguard0 -j ACCEPT
-          ${pkgs.iptables}/bin/iptables -t nat -A POSTROUTING -s 10.64.10.1/24 -o enp1s0 -j MASQUERADE
-          ${pkgs.iptables}/bin/iptables -t nat -A POSTROUTING -s 192.168.10.1/24 -o enp1s0 -j MASQUERADE
-          ${pkgs.iptables}/bin/ip6tables -A FORWARD -i wireguard0 -j ACCEPT
-          ${pkgs.iptables}/bin/ip6tables -t nat -A POSTROUTING -s fd02:f8eb:7ca4:5f4c::1/64 -o enp1s0 -j MASQUERAD
-          ${pkgs.iptables}/bin/ip6tables -t nat -A POSTROUTING -s dced:2718:5f06:718a::1/64 -o enp1s0 -j MASQUERADE
-          #${pkgs.iptables}/bin/iptables -t nat -I POSTROUTING -o enp1s0 -j MASQUERADE
-          #${pkgs.iptables}/bin/ip6tables -t nat -I POSTROUTING -o enp1s0 -j MASQUERADE
+          # Accept traffic from wireguard0
+          ${ip} -A FORWARD -i wireguard0 -j ACCEPT
+          # IPv4 rules
+          ${ip} -t nat -A POSTROUTING -s 10.64.10.1/24 -o enp1s0 -j MASQUERADE
+          ${ip} -t nat -A POSTROUTING -s 192.168.10.1/24 -o enp1s0 -j MASQUERADE
+          # IPv6 rules
+          ${ip6} -t nat -A POSTROUTING -s fd02:f8eb:7ca4:5f4c::1/64 -o enp1s0 -j MASQUERADE
+          ${ip6} -t nat -A POSTROUTING -s dced:2718:5f06:718a::1/64 -o enp1s0 -j MASQUERADE
```

Comment:
please move all those magic constants to some kind of config or declare then as named constants